File: PythonCode/1035.py
#Questão: Teste de Seleção 1
A, B, C, D = input().split()
A = int(A)
B = int(B)
C = int(C)
D = int(D)
Somacd = C + D
Somaab = A + B
Resto_a = A % 2
if B > C and D > A and Somacd > Somaab and Resto_a == 0 and C >= 0 and D >= 0:
    print("Valores aceitos")
else:
    print("Valores nao aceitos")

File: C++Code/1035.cpp
//Questão: Teste de Seleção 1
#include <iostream>

using namespace std;

int main(){
    int a,b,c,d,somacd,somaab,par;
    cin >> a>>b>>c>>d;
    somaab = a+b;
    somacd = c+d;
    par = a%2;

    if (b > c && d > a && somacd > somaab && c >=0 && d >=0 && par==0){
        cout <<"Valores aceitos" << endl;
    }
    else{
        cout <<"Valores nao aceitos"<< endl;
    }
    return 0;
}